### PR TITLE
Update tellstick.markdown

### PR DESCRIPTION
**Pull request by brange**
Add information about max_age

**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant)
https://github.com/home-assistant/home-assistant/pull/24249

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Review by frenck — APPROVED** [review]

**Comment by klaasnicolaas** [comment]
✅ Can be merged after merging the parent PR 👍

**Comment by frenck** [comment]
Closing PR, since the parent PR has been closed.

**Pull request closed without merging**
